src/Sarif.Driver/Sdk/Errors.cs

```diff
@@ -238,31 +238,34 @@ public static void LogExceptionLoadingPlugIn(string plugInFilePath, IAnalysisCon
         public static void LogUnhandledRuleExceptionAssessingTargetApplicability(
```

Comment:
We use the new stacks functionality to preserve stack details for issues that might require active investigation by a tool owner. We do not emit stack details for exceptions that indicate configuration errors.